src/main.c

```diff
@@ -60,15 +68,29 @@ static void setup_clock(void)
 }
 
 /**
+ * @brief Initial interruptions configuration.
+ *
+ * Interruptions enabled:
+ * - TIM1 Update interrupt.
```

Comment:
Maybe blank line before list.

Reply:
Yes, I will rebase this branch with current master.